.circleci/main.yml

```diff
@@ -1160,16 +1160,19 @@ workflows:
   jupyter-extension:
     when: << pipeline.parameters.run-jupyter-jobs >>
     jobs:
+      - build-docker-images
```

Comment:
hmm tricky here. since someone can likely run on same commit and push shad image. but guess thats find since its same commit same image.

Reply:
Not sure what you mean here?